### ISSUE TITLE
Help needed in assigning XML header with scan parameters from vendor scanner software

### ISSUE THREAD

**Issue by roshtha** [ismrmrd/ismrmrd#190]
Hello,

I am studying how to relate scan parameters in ISMRMRD XML header with respect to scan parameters specified in a vendor scanner software such as from Siemens for real scans. A screenshot from Siemens Syngo MRI software which I captured from a demonstration video in youtube is below:

![image](https://user-images.githubusercontent.com/5310634/153387366-3b6cda93-a541-4be2-803a-b80ee59fc395.png)

If I want to fill ISMRMRD header from scan parameters entered in scanner software user interface, basically I would like to know whether the values entered in scanner software’s user interface is assigned to ISMRMRD `reconSpace` matrix size and FoV. And later these values are used for determining `encodedSpace` matrix size and FoV by considering readout / phase oversampling and partial fourier factors. I would like to get expert advice on the correctness of below calculations for filling ISMRMRD XML header part with respect to scanner software fields. 

```
encoding.reconSpace.matrixSize.x = <Base resolution> 
encoding.reconSpace. matrixSize.y = <Base resolution> * (<Phase resolution %>/100) 
encoding.reconSpace.matrixSize.z = 1

encoding.reconSpace.fieldOfView_mm.x = <FoV read> 
encoding.reconSpace.fieldOfView_mm.y = <FoV read>*(<FoV phase %>/100)  
encoding.reconSpace.fieldOfView_mm.z = <Slice thickness> 

encoding.encodedSpace.matrixSize.x = reconSpace.matrixSize.x * Readout_oversampling_factor, if specified
encoding.encodedSpace.matrixSize.y = reconSpace. matrixSize.y * Phase_oversampling_factor, if specified
encoding.encodedSpace.matrixSize.z = 1

encoding. encodedSpace.fieldOfView_mm.x = reconSpace.fieldOfView_mm.x * Readout_oversampling_factor
encoding.encodedSpace.fieldOfView_mm.y = reconSpace.fieldOfView_mm.y * Phase_oversampling_factor
encoding.encodedSpace.fieldOfView_mm.z = <Slice thickness> 

encoding. encodingLimits. kspace_encoding_step_1. minimum = 0;
encoding. encodingLimits. kspace_encoding_step_1. maximum = (encoding.encodedSpace.matrixSize.y * partial_fourier_percentage)  – 1;
encoding. encodingLimits. kspace_encoding_step_1. center = (encoding. encodingLimits. kspace_encoding_step_1. maximum -  encoding. encodingLimits. kspace_encoding_step_1. minimum + 1) / 2;
```

Also want to know whether multiple `<encoding>`s can be used for representing parameters and data of different slice groups or is it handled through multiple ISMRMRD .h5 files?

Thanks All!

**Comment by roshtha** [comment]
@hansenms, Thanks for the reply. 

Sorry I did not mention about my actual requirement.

We want to develop a new UI software for our in-house developed MRI scanner with some basic sequences. 
As part of that, we are exploring the scanner software from different MR vendors to understand various scan properties to be captured and features to be incorporated.

We are using ISMRMRD in our software for representing scan parameters and raw data from our scanner.  
So far we have only dealt with fully sampled data from the scanner which we could successfully represent and reconstruct DICOM images and shows images in software UI. 
With fully sampled data, both encodedSpace and reconSpace tags were filled with the same values of Base Resolution and FoV that are captured through UI. 
The integration of localizer part for planning is only progressing. 
Now we want to consider different oversampling and partial fourier methods too.
 
We have already used ISMRMRD libraries in the scanner application and used `siemens_to_ismrmrd` coverter application many times and it was very helpful to test developed reconstruction algorithms with data from Siemens scanners. 

Now, I am not sure whether the Base Resolution, FoV and the other values MR operators are setting through localizer images are to be mapped to reconSpace or encodedSpace. 
I think they are part of reconSpace and matrix settings for raw data acquisition starts from there. 
Have gone through ISMRMRD paper and github documentations.  Still got confused in this part.

Thank You!
